spinn_machine/machine.py

```diff
@@ -52,14 +53,17 @@ class Machine(object):
     #  coordinates down the given link (0-5)
     LINK_ADD_TABLE = [(1, 0), (1, 1), (0, 1), (-1, 0), (-1, -1), (0, -1)]
 
-    BOARD_48_CHIPS = [
-        (0, 0), (0, 1), (0, 2), (0, 3), (1, 0), (1, 1), (1, 2), (1, 3), (1, 4),
-        (2, 0), (2, 1), (2, 2), (2, 3), (2, 4), (2, 5), (3, 0), (3, 1), (3, 2),
-        (3, 3), (3, 4), (3, 5), (3, 6), (4, 0), (4, 1), (4, 2), (4, 3), (4, 4),
-        (4, 5), (4, 6), (4, 7), (5, 1), (5, 2), (5, 3), (5, 4), (5, 5), (5, 6),
-        (5, 7), (6, 2), (6, 3), (6, 4), (6, 5), (6, 6), (6, 7), (7, 3), (7, 4),
-        (7, 5), (7, 6), (7, 7)
-    ]
+    CHIPS_PER_BAORD = {
```

Comment:
so many issues. 1. spelling. 2. why on earth put random cores when the only thing that uses it is the line below who cares about the keys? why not just keep what ti was before in this case?

Reply:
Spelling fixed.

The cores per chip is actually NOT random but reflects how the boards have actually been built.
These number reflect to 95% accuracy the cores for these chips on all boards.

Why the change is actually scope creep from a future idea of creating virtual machine that reflect the most likely configuration as accurately as possible.

Reply:
so scope creeps means reverting. given how you made me cherry pick everything . ;-)

but on what is the feature youve described. once you make a pr on that. ill move this comment over. but as i can now:

what on earth does generalizing over an entire machine what a board has per chip actually gain you!? is this part of my argument about getting the real big machine to begin with? coz if so. your path is another fudge upon a fudge. which is well.... fudgy.

Reply:
In practice every board is quite similar.
They all have the same chips with 17 core
So creating a virtual machine that reflects that will give you closer to a real machine then the correct solution of always using 17 or 18 cores on all chips so not a fudge at all.

Leaving here as it actually better describes the real world.